keras_cv/utils/bounding_box_test.py

```diff
@@ -114,3 +118,18 @@ def test_bounding_box_padding(self):
             ValueError, "Target shape should be larger than bounding box shape"
         ):
             bounding_box.pad_bounding_box_batch_to_shape(bounding_boxes, target_shape)
+
+    def test_mask_to_bbox(self):
+        target_boundingbox = [1, 1, 1, 1]
+        result = bounding_box.mask_to_bbox(self.center_box_mask_image)
+        self.assertAllClose(result, target_boundingbox)
+
+        self.center_box_mask_image_3d = tf.expand_dims(
```

Comment:
this can just be a variable, doesn't need to be assigned to `self`

Reply:
yes, by mistake.